include/lashup_kv.hrl

```diff
@@ -3,5 +3,6 @@
 -record(kv, {
```

Comment:
rather than extending the record in place, it probably makes sense to have a new record, like kv2